roboteam_ai/src/skills/InterceptBall.cpp

```diff
@@ -36,13 +37,23 @@ void InterceptBall::onInitialize() {
 }
 InterceptBall::Status InterceptBall::onUpdate() {
     ball = World::getBall();
-    //The keeper dynamically updates the intercept position as he needs to be responsive and cover the whole goal and this would help against curveballs e.g.
+    //The keeper dynamically updates the intercept position as he needs to be responsive and cover the whole goal and this would help against curveballs etc.
     if (keeper) {
         interceptPos = computeInterceptPoint(ball.pos,
                 Vector2(ball.pos) + Vector2(ball.vel)*constants::MAX_INTERCEPT_TIME);
     }
     deltaPos = interceptPos - robot->pos;
     checkProgression();
+    //interface
+    displayColorData.emplace_back(std::make_pair(interceptPos,Qt::red));
```

Comment:
these `emplace_back` 's be done in two lines by making a vector and by using displayColorData.insert

Reply:
Yes, but then all the information is in one line, I like this better because it is easier to read.